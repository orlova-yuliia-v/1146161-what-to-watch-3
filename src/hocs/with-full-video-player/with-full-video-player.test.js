import React from "react";
import renderer from "react-test-renderer";
import withFullVideoPlayer from "./with-full-video-player";

const MockComponent = () => <div />;

const MockComponentWrapped = withFullVideoPlayer(MockComponent);

it(`should render withFullVideoPlayer HOC correctly`, () => {
  const tree = renderer
  .create(
      <
        MockComponentWrapped
        autoPlay = {true}
        onExitButtonClick={() => {}}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
