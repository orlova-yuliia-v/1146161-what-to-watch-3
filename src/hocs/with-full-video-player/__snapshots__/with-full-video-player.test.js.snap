// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render withFullVideoPlayer HOC correctly 1`] = `<div />`;
